network/vpc/how-to/create-private-network.mdx

```diff
@@ -1,57 +1,108 @@
+
+Automatically generating a CIDR block is the default (and recommended) setting when creating a Private Network. Simply follow the steps [above](#how-to-create-a-private-network) to create a Private Network, leave **Advanced Settings** at default values, and a `/22` CIDR block will be automatically generated for the Private Network. 
+
+This block is guaranteed to be unique within the VPC, and will not overlap with any of your other Private Networks. It gives 1024 private IPv4 addresses, of which 1022 are available to be allocated to resources attached to the Private Network.
+
+#### Defining a custom CIDR block
+
+If you prefer, you can define your own custom IPv4 CIDR block when creating your Private Network. This is only recommended for advanced users.
+
+1. Follow the steps [above](#how-to-create-a-private-network) to create a Private Network, and at step 5 click **Advanced Settings**. 
+
+2. Click **Define custom**. The following screen displays:
+
+    TODO: insert screenshot
 
-4. Enter a **name** for the network, or leave the randomly-generated name in place. Optionally, you can also add one or many tags to help you organize your Private Networks. 
+3. Enter a network **prefix**. This should typically be in the following ranges normally used for private networks:
+    - `192.168.0.0` - `192.168.255.255`
+    - `172.16.0.0` - `172.31.255.255`
+    - `10.0. 0.0` - `10.255. 255.255`
 
-5. Click **Create regional Private Network** to finish. Your Private Network is created. You are taken back to the **regional Private Networks** dashboard, where your new regional Private Network is now displayed in the list.
+4. Choose your **network size** from the options available. The range of options begins at `/20`, with 2094 addresses available for resources on the Private Network, and ends at `/28`, with 14 addresses available for resources on the Private Network.
 
-## How to migrate an existing Private Network from zoned to regional
+5. Click **Create Private Network** to finish.
 
-Scaleway allows you to migrate an existing zoned Private Network, which is scoped to one [Availability Zone](/network/vpc/concepts/#regions-and-availability-zones) only, to make it a regional Private Network, scoped to an entire region.
+### Viewing IP addresses of attached resources
 
-This action must be carried out using the [Scaleway VPC API](https://www.scaleway.com/en/developers/api/). For full instructions, see our [dedicated documentation](/network/vpc/api-cli/vpc-private-beta/#migrating-existing-zonal-private-networks-to-regional-private-networks).
+When you attach resources to the Private Network, they will be assigned IP addresses from the CIDR block you defined. See [how to attach a resource to a Private Network](/network/vpc/how-to/attach-resources-to-pn/#how-to-attach-a-resource-to-a-private-network) and [how to view the resource's IP address](/network/vpc/how-to/attach-resources-to-pn/#how-to-view-the-resource's-ip-address) for more information.
```

Comment:
@RoRoJ the link works with an `'` in it? 
#how-to-view-the-resource**'**s-ip-address

(asking for curiosity)